nmigen_boards/resources/interface.py

```diff
@@ -102,7 +102,7 @@ def DirectUSBResource(*args, d_p, d_n, pullup=None, vbus_valid=None,
 
 
 def ULPIResource(*args, data, clk, dir, nxt, stp, rst=None,
-            clk_dir='i', attrs=None, conn=None):
+            clk_dir='i', invert_rst=False, attrs=None, conn=None):
```

Comment:
I'd say that if we have `clk_dir` we should also use `rst_invert` for consistency.

Reply:
I'm fine with that. Pushing that change.

(My original rationale was just following spoken order-- we "invert reset", and set the "clock direction", but I'm super fine with anything.)